.storybook/theme/SilverstripeTheme.js

```diff
@@ -3,7 +3,7 @@ import { create } from '@storybook/theming/create';
 const SilverstripeTheme = create({
   base: 'light',
   brandTitle: 'Silverstripe CMS - Storybook',
-  brandImage: 'https://www.silverstripe.com/_resources/themes/app/dist/images/logos/silverstripe-logo.svg',
+  brandImage: 'https://www.silverstripe.org/themes/ssv3/img/global-logo-open-source.svg',
```

Comment:
Does brandImage take a relative path? If so, I'd much rather just have the image here, either directly in this theme dir or in the .storybook/public/ dir, whichever works (if either works)

Reply:
Yep, you can include it from a local file. I have updated to include this, and also set a minimum width so the default sidebar width doesn't make it look too small.

<img width="524" alt="Screen Shot 2023-10-27 at 5 16 13 PM" src="https://github.com/silverstripe/silverstripe-admin/assets/415374/5da7d89d-12ca-48f5-bdf1-bc07d9154874">